src/luascript.cpp

```diff
@@ -10667,6 +10676,19 @@ int LuaScriptInterface::luaItemTypeCreate(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaItemTypeIsBlocking(lua_State* L)
+{
+	// itemType:isBlocking()
+	const ItemType* itemType = getUserdata<const ItemType>(L, 1);
+	if (itemType) {
+		pushBoolean(L, itemType->blockProjectile || itemType->blockSolid);
+	}
```

Comment:
@ranisalt unfortunately its not a comparison, is how this indentation style is named :rofl: